test/unit/builtins/array.spec.ts

```diff
@@ -487,6 +487,7 @@ test.each([
 });
 
 test.each([
+    { array: [[]], expected: [] },
```

Comment:
Could you also add testcase `{ array: [{ a: 1 }, { a: 2 }, { a: 3 }], expected: [{ a: 1 }, { a: 2 }, { a: 3 }] },` to this list (passes on current master)